tensorflow/core/util/gpu_kernel_helper.h

```diff
@@ -63,6 +63,39 @@ using gpuError_t = hipError_t;
 #define GetGPUStream(context) context->eigen_gpu_device().stream()
 
 namespace tensorflow {
+// Launches a GPU kernel through cudaLaunchKernel in Cuda environment, or
```

Comment:
CUDA instead of Cuda, but that shouldn't gate this PR.

Reply:
Appreciate this. Will update this as a follow-up PR mentioned in our conversation [here](https://github.com/tensorflow/tensorflow/pull/28568#discussion_r283381197)

Reply:
Can't beat the speed of a newly commited kernel `tridiagonal_matmul_op_gpu.cu.cc`. Did a force push to address the renaming here. As well as updating `Cuda` to `CUDA`. Now should be good.

This update serves as a good example to make me understand why a renaming needs to be updated in two phases.